pkg/eve/runme.sh

```diff
@@ -157,9 +157,17 @@ do_installer_raw() {
   dump "$OUTPUT_IMG" installer.raw
 }
 
+# create_installer_iso creates the installer iso and leaves it as /output.iso
+# common base for other usages like do_installer_iso and do_installer_net
+create_installer_iso() {
+  mkdir -p /installer_root
+  unsquashfs -f -d /installer_root /bits/installer.img 1>&2
+  tar -C /installer_root -cf - . | /make-efi installer
+  rm -rf /installer_root
+}
+
 do_installer_iso() {
-  rm -rf /parts
```

Comment:
That still leaves them coupled. I wonder if we should just make our own random directory in `make-efi` rather than `/parts`. Yeah, that is even cleaner. Why have this implicit interface?

Reply:
Take a look now @rene

Reply:
yeah, I agree... looks good now... thanks...